### PR TITLE
Adds The rest of the lipstick colours to cosmetics.dm

### PR DESCRIPTION
**Pull request by Moth-lantern**
## About The Pull Request
I noticed while working on a feature for a downstream, that three of the lipstick colours weren't actually seperate items, so this PR just adds those three so they can be spawned in by admins.
## Why It's Good For The Game
It lets admins spawn more colours of lipstick for costuming, also good for downstreams that might want to make lipsticks buyable.
## Changelog
:cl:
add: Adds the rest of the lipsticks to the cosmetics.dm
/:cl:

### CODE REVIEW

**Review by Time-Green — APPROVED** [review]

File: code/game/objects/items/cosmetics.dm [hunk 1]
```diff
@@ -49,6 +49,18 @@
...
+	name = "blue lipstick"
+	colour = "blue"
+
+/obj/item/lipstick/green
+	name = "green lipstick"
+	colour = "green"
+
+/obj/item/lipstick/white
+	name = "white lipstick"
+	colour = "white"
```
Comment (san7890) [inline code/game/objects/items/cosmetics.dm:62]: put these before `/obj/item/lipstick/black` so we don't ruin the nice congruity we have between that and `/obj/item/lipstick/black/death`. just cut-paste whatever you've added into line 47 and fix the spacings.

**Review by san7890 — CHANGES_REQUESTED** [review]

**Comment by Moth-lantern** [comment]
Addressed your comment and moved the lipsticks around, should keep the congruity you wanted

File: code/game/objects/items/cosmetics.dm [hunk 2]
```diff
@@ -44,11 +44,24 @@
...
 
 /obj/item/lipstick/black
 	name = "black lipstick"
 	colour = "black"
 
+
```
Comment (san7890) [inline code/game/objects/items/cosmetics.dm:64]:
```suggestion
```

Comment (san7890) [reply code/game/objects/items/cosmetics.dm]: we didn't need this newline. i just fixed it myself and this PR will merge soon

**Pull request merged**